### ISSUE TITLE
`process_data` and `static_callback` do not handle PaymentError

### ISSUE THREAD

**Issue by m-sanders** [mirumee/django-payments#119]
`payments.urls.process_data` and `payments.urls.static_callback` do not handle PaymentError and there appears to be no documented preferred method to catch these. For example when using Saleor with PayPal A declined card results in a 500. 

Am I supposed to implement and replace these views?

**Comment by patrys** [comment]
You are correct, this is something we've discovered about PayPal only recently. We need to handle this in the library.

**Comment by m-sanders** [comment]
There was this: https://github.com/mirumee/django-payments/pull/78 which seemed to catch it but it does not appear to have made it to master.

**Comment by m-sanders** [comment]
@patrys Do you have any recommendations as a temporary workaround for this issue or ideas on how you would like it fixed so I can submit a PR? Thanks.

**Comment by patrys** [comment]
I believe @mociepka has a working fix/workaround for this. Michał?

**Comment by vhf** [comment]
I ran into this issue today. Any news?

**Comment by BenSturmfels** [comment]
I'm suspecting this issue occurs when a payment is declined on paypal.com and the customer clicks the link to return to the originating site. Hoping to verify with an affected customer today. In any case, the above merge request #203 should solve this, though I'll post again after I know for sure.
